core/src/main/scala-2.13+/sttp.tapir/ModifyMacroSupport.scala

```diff
@@ -0,0 +1,17 @@
+package sttp.tapir
```

Comment:
is it github or is this file in the `sttp.tapir` directory? (not `sttp/tapir`)?

Reply:
Thank you IntellijIdea, great job :1st_place_medal: